### PR TITLE
Remove duplicate test code.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: tests/TestCase/Database/QueryTest.php [hunk 1]
```diff
@@ -2227,21 +2227,6 @@ public function testSelectDistinctON()
             [3, 1],
             collection($results)->sortBy('author_id')->extract('author_id')->toList()
         );
-
-        $query = new Query($this->connection);
```
Comment (ADmad) [inline tests/TestCase/Database/QueryTest.php]: The exact same query instance and assertions are done above. So unless I am missing something the removed code was redundant.

**Review by saeideng — CHANGES_REQUESTED** [review]
its not same 
see `->distinct('author_id')`  and `->distinct(['author_id'])`

**Comment by dereuromark** [comment]
An inline comment might have clarified this.

**Comment by markstory** [comment]
Closing for the reason @saeideng mentioned.

**Pull request closed without merging**